### PR TITLE
Readme fix

### PR DESCRIPTION
**Pull request by kernelold**
Readme fix for Scaffolding steps

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ $ k8t new cluster MyCluster
 Create a new environment
 
 ```bash
-$ k8t new environment staging
+$ k8t new environment staging -c MyCluster
```
Comment (AFriemann) [inline README.md:123]: actually both are possible, the idea being that you could globally define something for the staging environment as well as for the staging environment in a particular cluster

File: README.md [hunk 2]
```diff
@@ -120,7 +120,7 @@ $ k8t new cluster MyCluster
 Create a new environment
```
Comment (AFriemann) [inline README.md:120]:
```suggestion
Create a new global environment
```

File: README.md [hunk 3]
```diff
@@ -120,7 +120,7 @@ $ k8t new cluster MyCluster
...
 
 ```bash
-$ k8t new environment staging
+$ k8t new environment staging -c MyCluster
 ```
```
Comment (AFriemann) [inline README.md:125]:
```suggestion

And a new cluster environment

```bash
k8t new environment staging -c MyCluster
```

```

**Review by AFriemann — CHANGES_REQUESTED** [review]
what do you think about this for clarity?

File: README.md (hunk 1 above)
Comment (kernelold) [reply README.md:123]: TBH I feel we need kinda flow here

File: README.md [hunk 4]
```diff
@@ -117,12 +154,16 @@ Create a new cluster
 $ k8t new cluster MyCluster
 ```
 
-Create a new environment
+Create a new global environment
 
 ```bash
-$ k8t new environment staging
+$ k8t new environment staging -c MyCluster
```
Comment (AFriemann) [inline README.md:160]:
```suggestion
$ k8t new environment staging
```

File: README.md [hunk 5]
```diff
@@ -117,12 +154,16 @@ Create a new cluster
...
 ```
 
+And a new cluster environment
+
+```bash
+k8t new environment staging -c MyCluster
 Generate a new deployment template for cluster MyCluster (for a list of available templates see the `k8t new template --help`)
```
Comment (AFriemann) [inline README.md:168]: can't really comment this with a suggestion, but the inline code snippets are broken

File: README.md [hunk 6]
```diff
@@ -101,6 +101,43 @@ quickly verify the written templates.
 * `get_secret(key: str)` - provides a secret value from a given provider (see [here](#managing-secrets))
 * `bool(value: Any)` - casts value to boolean ("true", "on", "yes", "1", 1 are considered as `True`)
 
+## Overriding flow
+
+Overriding flow is from low score (1) to high (4) means more-specific configuration will be on top.
+
+So variables and templates will be ovveriden by flow `project` -> `environments` -> `clusters` -> `cluster-specific-environments`
+
+```.                                        (1) # k8t new project .
```
Comment (AFriemann) [inline README.md:110]: love this :+1:

File: README.md [hunk 7]
```diff
@@ -101,6 +101,43 @@ quickly verify the written templates.
 * `get_secret(key: str)` - provides a secret value from a given provider (see [here](#managing-secrets))
 * `bool(value: Any)` - casts value to boolean ("true", "on", "yes", "1", 1 are considered as `True`)
 
+## Overriding flow
+
+Overriding flow is from low score (1) to high (4) means more-specific configuration will be on top.
+
+So variables and templates will be ovveriden by flow `project` -> `environments` -> `clusters` -> `cluster-specific-environments`
```
Comment (AFriemann) [inline README.md:108]:
```suggestion
## Configuration inheritance

Configuration, values and templates are used according to the scope they are in. The following snippet shows an example project with low scores (1) and high scores (4) for evaluation order.

So variables and templates will be overridden from `project` -> `environments` -> `clusters` -> `cluster-environments` resulting in more specific configuration overriding lower values.
```

maybe sth like this?

**Review by AFriemann — CHANGES_REQUESTED** [review]

**Review by dcernag — APPROVED** [review]
nice!

**Review by AFriemann — APPROVED** [review]

**Pull request merged**